src/Globe/Globe.js

```diff
@@ -219,10 +220,18 @@ Globe.prototype.getZoomLevel = function( /*id*/ ) {
         };
 
     }();
+
     this.tiles.children[0].traverseVisible(cO);
     return cO();
 };
 
+
+Globe.prototype.computeDistanceForZoomLevel = function(zoom,camera) {
+
```

Comment:
3 hard-coded values in a single line: 2 (min level), 18 (max level), 6.0 (sse).

Proposal  : 
* `2` becomes `this.tiles.minLevel`
* `18` becomes `this.tiles.maxLevel`
* `6.0` : you could add a `export const SSE_SUBDIVISION_THRESHOLD` in `NodeProcess.js` and replace all usage of `6.0` in itowns: `NodeProcess.js`, `QuadTree.js` and here.

(and empty blank lines)